Workspace/src/xbox_controller/launch/state_select.launch

```diff
@@ -0,0 +1,23 @@
+<launch>
+	<node name="teleop_node" pkg="xbox_controller" type="state_select" />
+	
+	<node name="drive_lis_pub" pkg="xbox_controller" type="joy_to_twist" />
+	
+	<node name="joy_launcher" pkg="joy" type="joy_node">
+	
+	<param name="controller" type="string" value="/dev/input/by-id/usb-©Microsoft_Corporation_Controller_15D5CD4-joystick" />
```

Comment:
I think `name` has to be `"dev"` for this to work. Also just change the formatting a bit to make it a bit easier to read:
```
<node name="joy_launcher" pkg="joy" type="joy_node"> 
    <param name="dev" type="string" value="/dev/input/by-id/usb-©Microsoft_Corporation_Controller_15D5CD4-joystick" /> 
</node>